### ISSUE TITLE
Connect TVML client over internet

### ISSUE THREAD

**Issue by q20** [ggyeh/TVML-Kodi-Addons#1]
I have tested connecting to the service remotely by forwarding a single TCP port from my firewall to my NAS (5555, in my case), thinking this would work, but it doesn't. I think TVML says something like application.js not found... I don't quite remember and am not home to check. A port scan indicates the port is definitely listening...

**Comment by ggyeh** [comment]
These are the requirements:
1. You need to open the same port the app runs on (5000 by default).
2. Define in the client app (on the ATV) this ip:
IP=http://<ip of nas>:<port you defined>
Leave all rest as default.

The client app will try to reach: http://<ip of nas>:<port you defined>/js/application.js
Let me know if this works.

**Comment by q20** [comment]
Nah, I set the executable to use port 5555. This works locally. I then forward that port to my NAS, run a port scan from outside my network to ensure 5555 is listening and then try to connect TVML. Which doesn't work. Perhaps I can test again this evening and confirm settings, but I'm quite certain I've configured everything correctly.

**Comment by ggyeh** [comment]
Do you see something in the server console?
If the client reaches it, it should say something like:
GET /js/application.js HTTP/1.1" 200

If you don't see anything, there's something wrong in your connection.
